include/aspect/stokes_matrix_free.h

```diff
@@ -433,7 +436,10 @@ namespace aspect
       void evaluate_material_model() override;
 
       /**
-       * Add correction to system RHS for non-zero boundary condition.
+       * Add correction to system RHS for non-zero boundary condition. For more information
+       * on exactly what this correction is and why it is computed, see the deal.II tutorial
+       * step 50 section "LaplaceProblem::assemble_rhs()":
+       * https://www.dealii.org/developer/doxygen/deal.II/step_50.html#LaplaceProblemassemble_rhs
        */
```

Comment:
Same here..

Reply:
can you refer to the other location here, instead?